baseapp_payments/rest_framework/payment_methods/serializers.py

```
@@ -0,0 +1,71 @@
+        default_payment_method_id = validated_data.get("default_payment_method_id")
+        payment_method_id = validated_data.get("pk")
+        if default_payment_method_id:
+            try:
+                resp = stripe_service.update_customer(
+                    customer_id,
+                    invoice_settings={"default_payment_method": default_payment_method_id},
+                )
+                return resp
+            except Exception as e:
+                logger.exception(e)
+                raise serializers.ValidationError("Failed to update payment method")
```

Ruff (0.15.7)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)